File: ezSCORES/ezSCORES.Model/Teams.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model
{
	public  class Teams
	{
		public int Id { get; set; }
		public int UserId { get; set; }
		public string Name { get; set; }
		public int SelectionId { get; set; }
		public byte[]? Picture { get; set; }
		public virtual Users User { get; set; } = null!;
		public virtual Selections Selection { get; set; } = null!;
		public virtual ICollection<CompetitionsTeams> CompetitionsTeams { get; set; } = new List<CompetitionsTeams>();
	}
}

File: ezSCORES/UI/ezscores_mobile/lib/models/teams.dart
import 'package:ezscores_mobile/models/competitionsTeams.dart';
import 'package:ezscores_mobile/models/selections.dart';
import 'package:ezscores_mobile/models/users.dart';
import 'package:json_annotation/json_annotation.dart';
part 'teams.g.dart';
@JsonSerializable()
class Teams{
  int? id;
  String? name;
  String? picture;
  Selections? selection;
  Users? user;
  CompetitionsTeams? competitionTeam;

  Teams({this.id, this.name});

  factory Teams.fromJson(Map<String, dynamic> json) => _$TeamsFromJson(json);
  Map<String, dynamic> toJson() => _$TeamsToJson(this);
}